nominees/pepeketua-id-frog-identification.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Pepeketua ID - Frog identification",
+  "aliases": [
+    ""
+  ],
+  "description": "A pattern recognition software that facilitates the individual identification of frogs (Pepeketua) based on their skin marks.",
+  "website": "https://www.wildlife.ai/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/wildlifeai/pepeketua_landmarks/blob/main/LICENSE.txt"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 15,
+      "evidenceText": "This landmark detection model predicts six frog landmarks in an image. These landmarks can be used to morph and crop the image to facilitate the individual identification of each frog.",
+      "evidenceURL": "https://www.wildlife.ai/projects/pepeketua-id/"
```

Comment:
Could they provide more specific information on how this model/information contributes to SDG 15?

Reply:
+1

Reply:
Have done a follow-up on the above, will revert once we hear from the team. Thanks.